docs/src/css/custom.css

```
@@ -1,33 +1,150 @@
 /**
- * Any CSS included here will be global. The classic template
- * bundles Infima by default. Infima is a CSS framework designed to
- * work well for content-centric websites.
+ * Cleanuparr Documentation Theme
+ * Aligned with the frontend glassmorphism design system.
  */
 
-/* You can override the default Infima variables here. */
+/* ===== Fonts ===== */
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300..800&family=JetBrains+Mono:wght@400;700&display=swap');
```

Stylelint (17.7.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Inter:wght@300..800&family=JetBrains+Mono:wght@400;700&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Inter:wght@300..800&family=JetBrains+Mono:wght@400;700&display=swap'" (import-notation)

(import-notation)